text/chapter3.md

```diff
@@ -591,7 +591,7 @@ However, this chapter has also included examples of _infix_ [binary operators](h
 infix 4 eq as ==
 ```
 
-and therefore `entry.firstName == firstName` in `filterEntry` could be replaced with the `eq entry.firstName firstName`.
+and therefore `entry.firstName == firstName` in `filterEntry` could be replaced with the `eq entry.firstName firstName`. We'll cover a few more examples of defining infix operators later in this section.
```

Comment:
I imagine some readers getting hung-up on not fully understanding that infix line.